ui/codemods.log

```diff
@@ -0,0 +1,505 @@
+2020-09-24T17:50:46.692Z [warn] WARNING: {{capitalize}} was not converted as it has positional parameters which can't be automatically converted. Source: app/templates/partials/form-field-from-model.hbs
```

Comment:
I feel like we could probably delete this file, but @Monkeychip you know more about the codemod process than I do

Reply:
Yes, it's safe to delete. I'll go ahead and remove.